docs/source/overview/what-is-paraglider.rst

```diff
@@ -0,0 +1,30 @@
+.. _whatisparaglider:
+
+What is Paraglider?
+-------------------
+
+Paraglider is a control plane for cloud networking resources designed to simplify the tenant networking experience. 
+The Paraglider Controller exposes the Paraglider API to tenants and uses public cloud APIs to manage the tenant's cloud network.
+
+---
+
+How is Paraglider different from other cloud networking solutions?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Some existing options for creating/managing cloud networks are (1) using the underlying cloud APIs directly or (2) using something like Terraform.
+While Terraform has its benefits in being infrastructure as code, this doesn't solve the underlying problem: that we're still speaking in terms of low-level components like virtual networks, gateways, etc. 
+Other options in the space like service meshes can simplify connectivity between apps at the application layer, not the network layer. 
```

Comment:
Should we mention solutions like Aviatrix, inter cloud which create L3 overlay networks to create cloud connectivity?

Reply:
Today I was working on an email to circulate within my team about the open sourcing of Paraglider. I started with the text you wrote and morph it into something slightly different. Pasting here in case you want to cherry pick:

- Infrastructure as code solutions like Terraform might save you from directly invoking cloud provider APIs, and allow you to use a unified language to define your network, but everything is still expressed in terms of the resources and properties supported by each cloud, and you'll still assemble your network from low-level components like virtual networks, peering, gateways, etc. 
- Service meshes can simplify managing connectivity between services at the application layer, but often assume that someone else has set up the network-level connectivity for you.
- Paraglider also offers cloud-agnostic constructs, which it translates via a plug-in model, and transparently supports multi-region and multi-cloud connectivity, meaning that you use the same constructs whether you are establishing a connection within the same cloud and the same region, or across regions and across clouds.

Reply:
This is clean and nice. Will take!